README.md

```diff
@@ -28,27 +28,31 @@ module:
 
 ## How to compile AddOn code
 
-### C code/Scripts, with GRASS source code on your computer
+### C code/Scripts, with GRASS GIS core source code on your computer
```

Comment:
"C code/Scripts," probably no need to emphasize that in a heading, same further below.

Reply:
Addressed in 6ee0ec088fe70f62862ae9222f640bdd16578ffd